scripts/maintain.js

```diff
@@ -0,0 +1,20 @@
+console.log("Loading Maintains module!");
+maintain = document.getElementById("maintains-container");
+
+function setMaintain(number) {
+    let firstHalf = 100 - number;
```

Comment:
I think we need to divide the bar into three stages. Green, orange and red. Maybe: 100%-50% = green, 50%-25% = orange and 25%-0% = red?

Reply:
Yes exactly, 
I think we should make a separate issue for java-script maybe?

Reply:
Would be a good idea.

Reply:
I made a issue for it here: 
https://github.com/bierproductie/dashboard/issues/4

Reply:
Great

Reply:
Great.